libs/utils/utils.js

```diff
@@ -775,10 +775,12 @@ export async function loadIms() {
       reject(new Error('Missing IMS Client ID'));
       return;
     }
+    const unavMeta = getMetadata('universal-nav')?.trim();
+    const defaultScope = `AdobeID,openid,gnav${unavMeta && unavMeta !== 'off' ? ',pps.read,firefly_api' : ''}`;
     const timeout = setTimeout(() => reject(new Error('IMS timeout')), 5000);
     window.adobeid = {
       client_id: imsClientId,
-      scope: imsScope || 'AdobeID,openid,gnav,pps.read,firefly_api',
+      scope: imsScope || defaultScope,
```

Comment:
is this needed? we have two use-cases:
1. consumer does not override the scope: `getConfig()` will return the default Milo scope from https://github.com/adobecom/milo/pull/1947/files#diff-12979217e83810ba83417c3e7b030afd400eff7c83cd6390d9f9de3a337618ecR130
2. consumer sets a custom scope: `getConfig()` will return the scope set by the consumer.

@mokimo - do you remember why we added this fallback in the first place?

Reply:
The first use-case doesn't sound correct [this](https://github.com/adobecom/milo/pull/1947/files#diff-12979217e83810ba83417c3e7b030afd400eff7c83cd6390d9f9de3a337618ecR130) isn't the default scope - it's the milo specific scope. Consumers will never inherit anything in `scripts.js` nor load the milo core one at all.

So we do need the defaultScope in case a consumer doesn't supply one.

Reply:
@mokimo - thank you, that's correct! In that case it makes sense to keep the default scope, but maybe we can use the pattern described above - https://github.com/adobecom/milo/pull/1947#discussion_r1511032456.

Reply:
Hey @narcis-radu, the note section in the PR body was based on a previous conversation we had:
> **NOTE**: even though all consumers call setConfig in their scripts, as suggested in [Milo College](https://github.com/adobecom/milo-college/blob/main/scripts/scripts.js#L61), there aren't any defaults being set for IMS, as can be seen in the setConfig [method](https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L209-L236). Thus, it seems reasonable to me to adapt both files where scopes are declared.

It's the same as @mokimo mentioned, that consumers don't use the Milo-specific configs under any circumstance..

I also added a comment on why I think it's best to keep the config as a one-liner, to make it instantly clear what the possible IMS scopes are and what influences them.

Reply:
ok, we can go with that since the default `imsScope` does not change that often. we should also look into removing the default config value as you mentioned, but it can be a separate PR.

Reply:
Added a new commit to remove the config from scripts.js